connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -194,6 +200,13 @@ public Connector getConnector(String connType) {
         return instantiateConnector(connectorClass);
     }
 
+    public static Set<Class<? extends Connector>> getConnectorPlugins() {
```

Comment:
Is there any reason this needs to be in `Worker`? Seems like we could just do all of this in `AbstractHerder` and avoid introducing this extra API in `Worker`?